packages/config/config/devices/0x010f/templates/fibarogroup_templates.json

```diff
@@ -0,0 +1,294 @@
+{
+	"enable_disable_bit": {
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 1,
+		"defaultValue": 0,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Disable",
+				"value": 0
+			},
+			{
+				"label": "Enable",
+				"value": 1
+			}
+		]
+	},
+	"enable_disable_bit_reverse": {
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 1,
+		"defaultValue": 0,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Enable",
+				"value": 0
+			},
+			{
+				"label": "Disable",
+				"value": 1
+			}
+		]
+	},
+	"enable_disable_bit_default_true": {
+		"$import": "#enable_disable_bit",
+		"defaultValue": 1
+	},
+	"secure_assoc_bit": {
+		"$import": "#enable_disable_bit",
+		"defaultValue": 1,
+		"description": "This is only active when the node is included securely"
+	},
+	"enable_disable_all_on_off": {
+		"label": "Enable/Disable ALL ON/OFF",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 255,
+		"defaultValue": 255,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "ALL ON disabled / ALL OFF disabled",
+				"value": 0
+			},
+			{
+				"label": "ALL ON disabled / ALL OFF active",
+				"value": 1
+			},
+			{
+				"label": "ALL ON active / ALL OFF disabled",
+				"value": 2
+			},
+			{
+				"label": "ALL ON active / ALL OFF active",
+				"value": 255
+			}
+		]
+	},
+	"change_bi_stable_keys": {
+		"label": "Change [On-Off] bi-stable keys",
```

Comment:
Looks weird - is there a nicer way for this?

Reply:
To be honest I don't really know what it is. But it was used in two files so I broke it out here so if we change it we update both.